plugins/woocommerce/templates/myaccount/navigation.php

```diff
@@ -22,7 +22,7 @@
 do_action( 'woocommerce_before_account_navigation' );
 ?>
 
-<nav class="woocommerce-MyAccount-navigation">
+<nav class="woocommerce-MyAccount-navigation" aria-label="Account Pages">
```

Comment:
```suggestion
<nav class="woocommerce-MyAccount-navigation" aria-label="Account pages">
```

Nitpick: The majority of `aria-label` in our codebase are using sentence case. What are your thoughts on using this here as well?